macros/shared

```diff
@@ -15,8 +15,8 @@
   '-fexceptions', '-fstack-clash-protection', '-fno-semantic-interposition' \
   %{nil}}
 %_cross_cxxflags %_cross_cflags
-%_cross_ldflags -Wl,-z,relro -Wl,-z,now
-%_cross_c_link_args '-Wl,-z,relro', '-Wl,-z,now'
+%_cross_ldflags -Wl,-z,relro -Wl,-z,now -Wl,-Bsymbolic-functions
```

Comment:
I searched for the flag and it seems like it is used to improve library load times? Do you have a link to a blogpost or something like what you shared in other commits (the links to the fedora docs), that explains why this is a good addition to the list of flags? The man pages aren't that descriptive, and I'm not a `gcc/binutils` expert so I don't fully understand the impact of this or its benefits :sweat_smile: :

> tells the linker to bind references to global function symbols to the definition inside the shared library

Reply:
I shared these links in the description:
> There's some light background reading [here](https://maskray.me/blog/2021-05-16-elf-interposition-and-bsymbolic) and [here](https://maskray.me/blog/2021-05-09-fno-semantic-interposition).